src/base/searchengine.cpp

```diff
@@ -255,10 +257,12 @@ void SearchEngine::updateIconPath(PluginInfo * const plugin)
 
 void SearchEngine::checkForUpdates()
 {
-    // Download version file from update server on sourceforge
-    Net::DownloadHandler *handler = Net::DownloadManager::instance()->downloadUrl(m_updateUrl + "versions.txt");
-    connect(handler, SIGNAL(downloadFinished(QString, QByteArray)), this, SLOT(versionInfoDownloaded(QString, QByteArray)));
-    connect(handler, SIGNAL(downloadFailed(QString, QString)), this, SLOT(versionInfoDownloadFailed(QString, QString)));
+    // Download version file from update server
+    using namespace Net;
+    DownloadHandler *handler = DownloadManager::instance()->downloadUrl(m_updateUrl + "versions.txt");
+    connect(handler, static_cast<void (DownloadHandler::*)(const QString &, const QByteArray &)>(&DownloadHandler::downloadFinished)
```

Comment:
I already want to redo DownloadHandler to get rid of these overloaded signals, etc. Apparently, I'll do it soon, but for now the only way...